class/Services/Content/Passle/PassleContentServiceBase.php

```diff
@@ -89,8 +97,9 @@ public static function fetch_all_by_passle(string $passle_shortcode)
 
     $responses = static::get_all_paginated($url);
 
-    if (in_array(null, $responses)) {
-      throw new Exception("Failed to get data from the API", 500);
+    if (is_null($responses) || in_array(null, $responses)) {
+      // throw new Exception("Failed to get data from the API", 500);
```

Comment:
If the API Settings are incorrect, throwing an error leaves the Posts tab as disabled, rather than enabled but empty. This then also means that once you've corrected the API settings, you need to refresh the page rather than reloading the posts.

Reply:
Ah okay, an empty page sounds better than disabling the tab. Can we remove the commented line please?